src/test/java/org/reaktivity/nukleus/tcp/internal/streams/rfc793/ClientResetAndAbortIT.java

```diff
@@ -68,7 +68,6 @@
     @Rule
     public final TestRule chain = outerRule(SocketChannelHelper.RULE)
             .around(reaktor).around(counters).around(k3po).around(timeout);
-
```

Comment:
This blank line should not have been removed

Reply:
I've restored it. Please merge if it's ok.